src/main/java/com/dreammaster/gthandler/recipes/ElectrolyzerRecipes.java

```diff
@@ -8,293 +8,491 @@
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 2L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 6L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
                 378,
                 120);
+        // Glauconite Dust
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Glauconite, 13L),
-                ItemList.Cell_Empty.get(4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Glauconite, 21L),
+                ItemList.Cell_Empty.get(6L),
                 GT_Values.NF,
                 Materials.Hydrogen.getGas(2000L),
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 1L),
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 2L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 6L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
                 378,
                 120);
+        // Vermiculite
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Vermiculite, 21L),
-                ItemList.Cell_Empty.get(8L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Vermiculite, 37L),
+                ItemList.Cell_Empty.get(2L),
                 GT_Values.NF,
-                Materials.Hydrogen.getGas(2000L),
+                Materials.Hydrogen.getGas(10000L),
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Iron, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Water, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 12L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 2L),
+                GT_Values.NI,
                 GT_Values.NI,
                 null,
                 420,
                 120);
+        // Mica
```

Comment:
this does not work out. I think it should just be raw silicon, not dioxide.

Reply:
yes i forgot to change resource type